fastn-core/src/host_builtins.rs

```diff
@@ -57,5 +58,33 @@ pub fn app_path(pkg: &fastn_core::Package, req_path: &str) -> fastn_resolved::De
         js: None,
         line_number: 0,
         external_implementation: false,
-    })
+    });
+
+    (name, def)
+}
+
+/// Ftd string variable that holds the name of the package.
+///
+/// Useful to determine if the package is run standalone or as a dependency:
+#[inline]
+pub fn main_package(pkg: &fastn_core::Package) -> (String, fastn_resolved::Definition) {
```

Comment:
This should also take `fastn_core::Config` as argument, for the same reason I discussed yesterday.

Reply:
Also maybe you can make `impl Config { fn main_package() {} }` so it shows up in code completion etc, which has better discovery than scattered utility methods.